Tests/ApolloTests/RequestChainTests.swift

```diff
@@ -966,6 +966,7 @@ class RequestChainTests: XCTestCase {
 
   // MARK: `proceedAsync` Tests
 
+  @available(*, deprecated)
```

Comment:
Are these just for us to know what we should remove when finally removing the deprecated properties/functions?

Reply:
When you use a deprecated method in the test, it emits a warning. Making the test "deprecated" suppresses that warning, but the tests still run just fine :)

Reply:
That's an odd quirk of testing. Useful though - thanks.